code/modules/modular_computers/NTNet/NTNRC/conversation.dm

```diff
@@ -70,10 +80,8 @@ var/global/ntnrc_uid = 0
 			changeop(newop)
 
 	for(var/datum/computer_file/program/chatclient/CC in clients)
-		if(CC == C || !CC.computer.screen_on)
-			continue
-		if(CC.computer.active_program == CC || (CC in CC.computer.idle_threads))
-			CC.computer.output_message(FONT_SMALL("<b>([title]) A client ([C.username]) has left the chat.</b>"), 0)
+		if(CC.program_state > PROGRAM_STATE_KILLED && CC != C)
+			CC.computer.output_message(FONT_SMALL("<b>([get_title(CC)]) A client ([C.username]) has left the chat.</b>"), 0)
```

Comment:
This sentence flows really weird. Maybe all of these should be "C has entered the chat."/"C has left the chat." instead.